### PR TITLE
Install blackbox exporter as part of out monitoring stack

### LINKED ISSUE #218: Set up prometheus-blackbox-exporter in Ansible
Add a new role, named `prometheus-blackbox-exporter` (the
`prometheus-node-exporter` role can be taken as a guideline), and adjust our
playbook to install it on lovelace as part of the monitoring play.

Next up, adjust the Prometheus configuration (found in the Prometheus role
defaults) to scrape the blackbox exporter.

The `prometheus-blackbox-exporter` role should also allow configuration of the
blackbox exporter. For this, copy the configuration file from the current
distribution default, place it in the `templates` directory of the role, and
`template:` it over as part of the role. The exporter should be restarted on
changes. We might want to customize this such that an SSH access check can be
used.

### PR DESCRIPTION
**Pull request by shtlrs**
Closes #218

Prometheus' defaults are deleted since we rely on hostvars

### CODE REVIEW

File: ansible/roles/prometheus/defaults/main.yml [hunk 1]
```diff
@@ -43,5 +43,41 @@ prometheus_configuration:
...
         - targets: ['localhost:9100']
 
+    - job_name: blackbox-ssh
+      metrics_path: /probe
+      params:
+        module: [ssh_banner]
+
+      static_configs:
+        - targets:
+            - lovelace.box.pydis.wtf
```
Comment (jchristgit) [inline ansible/roles/prometheus/defaults/main.yml:53]: Wanna add turing here as well?

File: ansible/roles/prometheus/defaults/main.yml [hunk 2]
```diff
@@ -43,5 +43,41 @@ prometheus_configuration:
...
+        - target_label: __address__
+          replacement: localhost:9115
+
+    - job_name: blackbox-http
+      metrics_path: /probe
+      params:
+        module: [http_2xx]
+      static_configs:
+        - targets:
+            - https://pydis.wtf/
```
Comment (jchristgit) [inline ansible/roles/prometheus/defaults/main.yml:74]: We should also add https://cloud.native.is.fun.and.easy.pydis.wtf to the list

**Review by jchristgit — COMMENTED** [review]
The Mind Toils Restlessly

File: ansible/roles/prometheus/defaults/main.yml (hunk 1 above)
Comment (shtlrs) [reply ansible/roles/prometheus/defaults/main.yml]: Done

File: ansible/roles/prometheus/defaults/main.yml (hunk 2 above)
Comment (shtlrs) [reply ansible/roles/prometheus/defaults/main.yml]: Granted

**Review by jchristgit — APPROVED** [review]
Looks good!

**Comment by jchristgit** [comment]
Thanks!

**Pull request merged**